tests/conftest.py

```diff
@@ -149,12 +145,53 @@ class User(db.Document, UserMixin):
 
 
 @pytest.fixture()
-def sqlalchemy_datastore(request, app, tmpdir):
-    from flask_sqlalchemy import SQLAlchemy
```

Comment:
We need to keep the `flask_sqlalchemy` in the tests. Could you add another fixture for the "pure" sqlalchemy version?